CONTRIBUTING.md

```diff
@@ -0,0 +1,47 @@
+# CONTRIBUTING
+
+## Markdown Guidlines
+
+Although we use the MDX format which sometimes allows the instantiation of React components in mdx files (see the [Readme section on this subject](/README.md#MDX)), it is recommended to avoid doing so to facilitate the contribution.
```

Comment:
./README I think

Reply:
I meant `./README.md#MDX` instead of `/README.md#MDX`. with the leading `.`

Reply:
it's working without the `.`

Reply:
ok sorry then